api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -110,6 +114,20 @@ public void setUserProvider(Provider<User> userProvider) {
       fcCluster = this.notebooksService.createCluster(
           project, NotebooksService.DEFAULT_CLUSTER_NAME, userProvider.get().getEmail());
     }
+    if (org.pmiops.workbench.notebooks.model.ClusterStatus.ERROR.equals(fcCluster.getStatus())) {
+      User user = this.userProvider.get();
```

Comment:
Actually - should update this logic to break if we exceed 2 retries. Currently this would just keep incrementing cluster retries indefinitely on every poll.